### PR TITLE
Added README Debug Variables Section

### PR DESCRIPTION
**Pull request by sharmag1-Gaurav**
Took inspiration from the following link.
https://www.intel.com/content/www/us/en/support/programmable/support-resources/design-guidance/opencl-software-support.html#Debug:~:text=3.-,Debug,-Emulation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -236,6 +236,14 @@ or IntelⓇ FPGA RTE for OpenCL™ installed on your system:
 
 6.  Run your OpenCL host program.
 
+
+### Debug Environment Variables
+
+| Environment Variable Title  | Description |
+| ------------- | ------------- |
```
Comment (zibaiwan) [inline README.md:243]: Can you please add a description for `ACL_DEBUG` environment variable? It's just a general runtime debug env var, basically output everything else other than `ACL_HAL_DEBUG` and `ENV_ACL_CONTEXT_CALLBACK_DEBUG`. You can search for the code for a brief use case.

**Review by zibaiwan — CHANGES_REQUESTED** [review]
Thanks Gaurav, this looks good to me. Just a minor change requested.

File: README.md (hunk 1 above)
Comment (sharmag1-Gaurav) [reply README.md:243]: Added ACL_DEBUG, please look it over when you can :)

**Comment by zibaiwan** [comment]
I will merge once the CI finishes.

**Pull request merged**